src/components/functions.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+export default  function optimizeSvg(dataUrl){
+  return axios.post(`http://localhost:3500/optimizeSvg`, { dataUrl })
```

Comment:
cant you use a variable for http://localhost:3500/ , and define it in one place and use it throughout.